index.d.ts

```diff
@@ -276,7 +275,7 @@ declare class Shopify {
     payment: {
         count: (checkoutToken: any) => Promise<number>;
         create: (checkoutToken: any, params: any) => Promise<any>;
-        get: (checkoutToken: any, id: number) => Promise<any>;
+        get: (checkoutId:number, checkoutToken: any, id: number) => Promise<any>;
```

Comment:
This doesn't look correct, I think it shouldn't be changed, see https://help.shopify.com/en/api/reference/sales_channels/payment#show

Reply:
Right, somehow missed the id...

BTW, `checkoutToken` should be a `string`, right or is there a reason it's `any` right now?

Reply:
Yes you are right, I think there is no particular reason, thanks for fixing it.